site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -471,6 +471,63 @@ To ensure that the ecosystem has sufficient time to test, we don’t expect to r
 
 {% endDetails %}
 
+{% Details %}
+{% DetailsSummary %}
+### How is first-party data used in a Protected Audience auction? 
+{% endDetailsSummary %}
+
+First-party data is data owned by the site on their users. For example, if a user has specified their favorite color on the advertiser’s or publisher’s site, that color is considered first-party data. 
+
+In a Protected Audience auction, the advertiser can use their first-party data to determine the [ad interest group membership](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#11-joining-interest-groups), and can also pass data into the interest group as [`userBiddingSignals`](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#12-interest-group-attributes). The first-party data from the advertiser will be available only for the buyers during the bid generation step, and is not available for the sellers.  
+
+For example, if the advertiser knows the user’s favorite color, the value can be passed in when the user is added to an interest group: 
```

Comment:
I guess the way this sentence is written, I expect the value to be passed in "something"?

Reply:
good call.  i've updated the sentence.